Configurations/unix-Makefile.tmpl

```diff
@@ -912,12 +912,14 @@ EOF
 # It's not necessary to have both as targets.  The choice falls on the
 # simplest, {libname}$shlibextimport for Windows POSIX layers and
 # {libname}$shlibextsimple for the Unix platforms.
+# The AIX targets in Makefile.shared need SHLIB_MAJOR and SHLIB_MINOR.
 $target: $lib$libext $deps $ordinalsfile
 	\$(MAKE) -f \$(SRCDIR)/Makefile.shared -e \\
 		PLATFORM=\$(PLATFORM) \\
 		PERL="\$(PERL)" SRCDIR='\$(SRCDIR)' DSTDIR="$libd" \\
 		INSTALLTOP='\$(INSTALLTOP)' LIBDIR='\$(LIBDIR)' \\
 		LIBDEPS='\$(PLIB_LDFLAGS) '"$linklibs"' \$(EX_LIBS)' \\
+		SHLIB_MAJOR='\$(SHLIB_MAJOR)' SHLIB_MINOR='\$(SHLIB_MINOR)' \\
 		LIBNAME=$libname SHLIBVERSION=\$(SHLIB_MAJOR).\$(SHLIB_MINOR) \\
```

Comment:
Did you notice `SHLIBVERSION` here?  I don't see a need to add another two variables...